packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -110,6 +125,30 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
     });
     this.searchStatus = buildSearchStatus(this.bindings.engine);
     this.generatedAnswerCommon.insertFeedbackModal();
+
+    if (window.ResizeObserver) {
```

Comment:
Should we only create the resize observer when this.collapsible is true?

```suggestion
    if (window.ResizeObserver && this.collapsible) {
```

Reply:
indeed very good point!